posthog/models/session_recording_playlist_item/session_recording_playlist_item.py

```diff
@@ -1,13 +1,21 @@
 from django.db import models
 
+from posthog.models.team import Team
+
 
 class SessionRecordingPlaylistItem(models.Model):
-    session_id: models.CharField = models.CharField(max_length=200)
+    class Meta:
+        unique_together = ("recording_id", "playlist_id", "team")
+
+    recording: models.ForeignKey = models.ForeignKey(
+        "SessionRecording", related_name="playlist_items", on_delete=models.CASCADE, null=True
```

Comment:
I think we should link this to the `session_id` field
```suggestion
        "SessionRecording", related_name="playlist_items", on_delete=models.CASCADE, null=True, to_field="session_id"
```